assets/js/index.js

```diff
@@ -11,3 +11,12 @@ $('.typeahead').each(function applyTypeahead() {
     selectElement: this
   });
 });
+
+$(function () {
+  if ($('#loader-container')) {
+    $('.govuk-button').on('click', function () {
+      $('#loader-container').addClass('spinner-loader');
+      $('#report-submit').addClass('visuallyhidden');
+    });
+  }
+});
```

Comment:
Can we avoid using jQuery, so we can deprecate it from this project?

Reply:
removed jQuery functions and made changes with equivalent javascript functions.